tests/next_tests/integration_tests/feature_tests/ffront_tests/test_math_builtin_execution.py

```diff
@@ -116,6 +116,9 @@ def make_builtin_field_operator(builtin_name: str):
 
 @pytest.mark.parametrize("builtin_name, inputs", math_builtin_test_data())
 def test_math_function_builtins_execution(cartesian_case, builtin_name: str, inputs):
+    if cartesian_case.backend is None:
```

Comment:
wouldn't it be even simpler to generate a string and load it? I.e. reverse the strategy?

Reply:
From a global perspective simpler for sure. Not sure that's the right approach though. Since we didn't want to fix this here anyway, I'll think about it.